v2/datastream-to-spanner/terraform/samples/mysql-end-to-end/variables.tf

```diff
@@ -29,17 +29,18 @@ variable "datastream_params" {
     stream_prefix_path            = optional(string, "data")
     pubsub_topic_name             = optional(string, "live-migration")
     stream_id                     = optional(string, "mysql-stream")
-    max_concurrent_cdc_tasks      = optional(number, 50)
-    max_concurrent_backfill_tasks = optional(number, 50)
-    mysql_databases = list(object({
+    max_concurrent_cdc_tasks      = optional(number, 5)
+    max_concurrent_backfill_tasks = optional(number, 15)
```

Comment:
Any reasoning behind selecting 15 as the backfill tasks ?

Reply:
Is the default in Datastream. We had made a similar change to the other sample previously. 
5 for CDC and 15 for backfill are the defaults.

Reply:
I was under the impression 8 was the default and 50 is what we set for max parallelism (considering concurrent backfill). Let's just make sure the default being set here is being set based on some benchmarks. You can take this up in a follow up PR if required.

Reply:
I confused your comment with using Datastream defaults, will change this all of this to 5 and 50 (as per HB defaults) as part of the sharding PR.